src/Controller/ArticlesController.php

```diff
@@ -857,7 +857,7 @@ private function articlePageArguments(Request $request, string $id, int $version
                     $infoBars[] = new InfoBar('Read the <a href="'.$this->generatePath($history).'">most recent version of this article</a>.', InfoBar::TYPE_MULTIPLE_VERSIONS);
                 }
 
-                if ($item instanceof ArticlePoA) {
+                if ($item instanceof ArticlePoA && $latestVersion < 2) {
```

Comment:
This will only display `Accepted manuscript, PDF only. Full online edition to follow.` when there is only one version of the article and that version is PoA. We will still want to display this message for articles that have multiple versions but have not yet become version of record.